ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -305,10 +305,23 @@ class PartCentredForceHolder {
       string expected_version = "1.7.3, 1.7.2, 1.7.1, 1.7.0, 1.6.1, and 1.5.1";
 #endif
       string message = $@"Unexpected KSP version {Versioning.version_major}.{
-            Versioning.version_minor}.{Versioning.Revision}; this build targets {
-            expected_version}.";
-      if (GameDatabase.Instance.GetAtMostOneNode(
-              principia_override_version_check_config_name_) == null) {
+            Versioning.version_minor}.{
+            Versioning.Revision}; this build targets {expected_version}.";
+      bool fail_version_check = true;
+      ConfigNode override_version_check =
+          GameDatabase.Instance.GetAtMostOneNode(
+              principia_override_version_check_config_name);
+      if (override_version_check != null) {
+        foreach (var version in override_version_check.GetValues("version")) {
+          if (version ==
+              $@"{Versioning.version_major}.{Versioning.version_minor}.{
+                Versioning.Revision}") {
+            fail_version_check = false;
```

Comment:
This feels a bit too double-negativeish, and in both cases we failed the version check; we are just making it non-fatal.
`version_check_overriden`?

Reply:
Done.